docs/reference/templating/index.md

```diff
@@ -0,0 +1,36 @@
+---
+description: In many places Bifröst can be customized using templating. How does they work?
+---
+
+# Templating
+
+In many places of the [configuration](../configuration.md) Bifröst can be not just customized using static strings or numbers, also templates can be used.
+
+Bifröst is using the [template engine of Go](https://pkg.go.dev/text/template). Here is a collection of quite good resources to learn can be found at [HashiCrop's Nomand Developer documentation](https://developer.hashicorp.com/nomad/tutorials/templates/go-template-syntax).
```

Comment:
Bifröst uses the [template engine of Go](https://pkg.go.dev/text/template). A collection of rather good resources to learn can be found at [HashiCrop's Nomand Developer documentation](https://developer.hashicorp.com/nomad/tutorials/templates/go-template-syntax).